model.py

```diff
@@ -49,84 +27,65 @@ def _addContent(self, text):
     #Add GENERAL Contents
     def addGeneral(self, lines):
         if(lines):
-            self._addContent(r'''
-        <div class="Container RepeatY BGStyle{}">
-            <div class="Inner">
-                <h3>General</h3>'''.format(int(self._bgStyle)))
-
-            self._addContent(r'''
+            self._addContent(self._template_dictionary['OPEN_GENERAL'] \
```

Comment:
__[flake8]__
* [E502] the backslash is redundant between brackets

<!-- Comment by [SideCI](https://sideci.com) -->